index.js

```diff
@@ -0,0 +1,69 @@
+$(document).ready(function() {
+$('#load-trips').click(function(event) {
+  $.get('https://trektravel.herokuapp.com/trips',
+  response => {
+    console.log('success!');
+    console.log(response);
+    for (let i = 0; i < response.length; i++) {
+      $('#trips').append(`<article id='trip-${response[i].id}'><button class="dropdown button small-12">${response[i].name}</button></article>`);
+
+      $(`#trip-${response[i].id} > button`).click(function(event) {
+        console.log('you clicked');
+        if (window.$(`#info-${response[i].id}`)[0])  {
+          $(`#info-${response[i].id}`).remove();
+        } else {
+        $.get(`https://trektravel.herokuapp.com/trips/${response[i].id}`,
+        trip => {
+          $(`#trip-${trip.id}`).append(`<section class="row"><section id='info-${trip.id}' class='panel small-10 small-centered columns'><p>ID: ${trip.id}</p>
+            <p>NAME: ${trip.name}</p>
+            <p>CONTINENT:${trip.continent}</p>
+            <p>COST: ${trip.cost}</p>
+            <p>CATEGORY: ${trip.category}</p>
+            <p>WEEKS: ${trip.weeks}</p>
+            <p>ABOUT: ${trip.about}</p>
+            <form action='https://trektravel.herokuapp.com/trips/${trip.id}/reservations' method='post' id='form-${trip.id}'>
+            <section>
+              <label>Name</label>
+              <input type="text" id="name" placeholder="identification required"/>
+            </section>
+            <section>
+              <label>Email</label>
+              <input type="text" id="email" placeholder="electronic document transmission"/>
+            </section>
+            <section>
+              <label>Age</label>
+              <input type="text" id="email" placeholder="age in relation to orbital period of Earth around solar body"/>
+            </section>
+            <section>
+              <button type="submit" class="button round">Reserve a Spot for ${trip.name}</button>
+            </section>
+            </form>
+            </section>
+            </section>`);
+            $('body').on('submit', 'form', function (e){
```

Comment:
this event handler doesn't need to live inside the click handler of the `#trip-${response[i].id} > button` element. And that handler doesn't need to live in the click handler of the `#load-trips` element! You can use event delegation so you can define it before these elements are made in the DOM.

A bunch of stuff doesn't! But this is a concrete one.